test/test_corner_cases.cc

```diff
@@ -175,6 +175,66 @@ TEST(FailTest, DISABLED_E14) {
       {I(10)});
 }
 
+TEST(FailTest, E15){
```

Comment:
I'm confused - are these test intended to fail? They're called FailTest

Reply:
oh, I forgot to change that form an earlier version of those tests where the entire file was only failures. I will update the name.